pkg/operation/botanist/controlplane.go

```diff
@@ -510,6 +530,28 @@ func (b *Botanist) DeploySeedLogging() error {
 				"schedule": fmt.Sprintf("%d 0,6,12,18 * * *", ct.Minute()%54+5),
 				"suspend":  b.Shoot.IsHibernated,
 			},
+			"sgUsername": "curator",
+			"sgPassword": string(sgCuratorPassword),
+		},
+		"searchguard": map[string]interface{}{
+			"enabled": true,
```

Comment:
This should not be hard coded, but a flag or a feature gate to the gardener controller manager.

Reply:
You could  ignore my comment about this line.
I have misunderstood what does `..searchguard.enabled` mean.